Purchases/Misc/Deprecations.swift

```diff
@@ -0,0 +1,119 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  Deprecations.swift
+//
+//  Created by Nacho Soto on 11/15/21.
+
```

Comment:
let's add a comment explaining why we're keeping these methods but with empty implementations, so devs running into this understand that no one is expected to run into the fatalErrors because the methods are obsoleted.